google/cloud/pubsub/publisher_connection_test.cc

```diff
@@ -363,13 +364,20 @@ TEST(MakePublisherConnectionTest, TracingEnabled) {
   auto response =
       publisher->Publish({MessageBuilder{}.SetData("test-data-0").Build()})
           .get();
+  publisher->Flush({});
```

Comment:
Do we need to wait until the `Flush()` operation completes?  Or at least `response` is "ready" before testing what the spans contain?